runtime/onert/core/include/compiler/Compiler.h

```diff
@@ -119,12 +122,6 @@ class Compiler
    */
   void enableToFp16();
 
-  /**
-   * @brief   Set backends from string-encoded mappings from operation index to backend type (cpu,
-   * acl_cl)
-   */
-  void set_backend_from_str(const char *backend_settings);
```

Comment:
`set_backend_from_str` is no longer the member of class `Compiler`. It is the property of `CompilerOptions`.